mappings/net/minecraft/client/render/Camera.mapping

```diff
@@ -46,3 +47,4 @@ CLASS net/minecraft/class_4184 net/minecraft/client/render/Camera
 	METHOD method_19335 getHorizontalPlane ()Lnet/minecraft/class_1160;
 	METHOD method_19336 getVerticalPlane ()Lnet/minecraft/class_1160;
 	METHOD method_19337 reset ()V
+	METHOD method_23767 getRotationDirection ()Lnet/minecraft/class_1158;
```

Comment:
```suggestion
	METHOD method_23767 getRotation ()Lnet/minecraft/class_1158;
```
Since a quaternion already represents a rotation around an axis in a 3d-space, no need for `direction`.

Reply:
How does the quaternion represent the axis? When I look at it, all I can really see is the degrees of rotation.

Reply:
https://en.m.wikipedia.org/wiki/Quaternions_and_spatial_rotation

Reply:
```
When used to represent an orientation (rotation relative to a reference coordinate system), they are called orientation quaternions or attitude quaternions. 
```

Any votes for renaming this method and the field to `getOrientation()` (as it _is_ the Camera class).